js/client/modules/@arangodb/process-utils.js

```diff
@@ -1082,7 +1082,12 @@ function checkArangoAlive (arangod, options) {
   const ret = res.status === 'RUNNING' && crashUtils.checkMonitorAlive(ARANGOD_BIN, arangod, options, res);
 
   if (!ret) {
-    print(Date() + ' ArangoD with PID ' + arangod.pid + ' gone:');
+    if (!arangod.hasOwnProperty('message')) {
+      arangod.message = '';
+    }
+    let msg = ' ArangoD of role [' + arangod.role + '] with PID ' + arangod.pid + ' is gone';
+    print(Date() + msg + ':');
+    arangod.message += (arangod.message.len === 0) ? '\n' : '' + msg + ' ';
```

Comment:
Isn't it `length` in JavaScript?
```suggestion
    arangod.message += (arangod.message.length === 0) ? '\n' : '' + msg + ' ';
```